src/botPage/bot/Interface/ToolsInterface.js

```diff
@@ -1,13 +1,38 @@
 import CandleInterface from './CandleInterface';
 import MiscInterface from './MiscInterface';
 import IndicatorsInterface from './IndicatorsInterface';
+import { translate } from '../../../common/i18n';
 
 // prettier-ignore
 export default Interface => class extends IndicatorsInterface(
     MiscInterface(CandleInterface(Interface))) {
     getToolsInterface() {
         return {
-            getTime: () => parseInt(new Date().getTime() / 1000),
+            getTime   : () => parseInt(new Date().getTime() / 1000),
+            toDateTime: (timestamp) => {
+                let dateTime;
```

Comment:
You can change `let` into `const` as the value of `dateTime` is not changing. You can the move `const dateTime` into the `if`-statement to ensure the variable is only created when `timestamp` is a `number`.